app/javascript/components/public_health/PatientsTable.js

```diff
@@ -519,6 +519,9 @@ class PatientsTable extends React.Component {
     return `Monitoree ${rowData.name}`;
   };
 
+  // Adds the className `selected-row` to those rows who have been selected with the Bulk Action checkcbox
+  getRowClassName = rowData => (this.state.selectedPatients.filter(x => this.state.table.rowData[Number(x)].id === rowData.id).length ? 'selected-row' : null);
```

Comment:
did prettier style this? I kind of wish it was multiple lines just for readablility

Reply:
or do it inline in the custom table props?

Reply:
Fixed.